content/05.main-text.md

```diff
@@ -236,14 +244,39 @@ The table includes the following columns:
 
 ### Enriched paths
 
+TODO: write this section
+
+The paths webapp panel includes the following information (Figure {@fig:webapp}C):
+<!-- https://github.com/greenelab/connectivity-search-frontend/blob/63bb4acbcebe4b346882754719071856b30b43ba/src/definitions.json -->
+
+- **path**:
+  The sequence of edges in the network connecting the source node to the target node.
+  Duplicate nodes are not permitted in paths.
+- **path score**:
+  A metric of how meaningful the path is in describing the connectivity between the source and target node.
+  The score combines the magnitude of the metapath's p-value with the percent of the DWPC contributed by the path.
+- **% of DWPC**
+  The contribution of the path to the DWPC for its metapath.
+  This metric compares the importance of all paths of the same metapath from the source node to the target node.
+
+TODO: discuss path score and ranking by path score.
+
 ### Comparison to Rephetio
 
-### Detecting Mechanisms of Action for Indications
+TODO: write this section.
 
-Assess ability to predict paths in <https://github.com/SuLab/DrugMechDB>
+<!--
+  https://github.com/dhimmel/connectivity-search-manuscript/commit/8f5a2ae4732249d4b6f33a6a142cc4043152c458
+-->
```

Comment:
Thinking comparison to Rephetio would be a good section for me to tackle next. I think I can draft this section by Jan 3. Will require touching up some existing visualizations.